### PR TITLE
(WIP) adds xyz for samsung smartthings M6001_MPP01

### PR DESCRIPTION
**Pull request by orrpan**
I'm missing something

In the log:
`Received Zigbee message from 'Hall Farstu', type 'attributeReport', cluster 'manuSpecificSamsungAccelerometer', data '{"x_axis":41,"y_axis":-1073,"z_axis":17}' from endpoint 1 with groupID 0`

On the device:
image

### CODE REVIEW

**Comment by Koenkk** [comment]
Is this still wip or can it be merged?

**Comment by orrpan** [comment]
The exposes does not work, but otherwise it mergeable

**Comment by Koenkk** [comment]
Because the value stays `N/A`? Can you provide a screenshot of the State tab?

**Comment by orrpan** [comment]
yessir
image

**Comment by Koenkk** [comment]
I don't see any axis here (e.g. `x_axis`), does this device actually support it?

**Comment by ssmall** [comment]
I tested this change locally and it worked for me. I see the following in HomeAssistant after reconfiguring the device:
Screenshot 2021-02-27 215601

File: devices.js [hunk 1]
```diff
@@ -8216,7 +8216,7 @@ const devices = [
         model: 'IM6001-MPP01',
         vendor: 'SmartThings',
         description: 'Multipurpose sensor (2018 model)',
-        fromZigbee: [fz.temperature, fz.ias_contact_alarm_1, fz.battery, fz.smartthings_acceleration],
+        fromZigbee: [fz.temperature, fz.battery, fz.ias_contact_alarm_1, fz.smartthings_acceleration],
```
Comment (ssmall) [inline devices.js:8219]: is this change necessary? I was able to get accelerometer data to report without changing this line at all.

**Comment by orrpan** [comment]
> I tested this change locally and it worked for me. I see the following in HomeAssistant after reconfiguring the device:
> Screenshot 2021-02-27 215601

Nice! Then my setup just has some strange hiccup.

File: devices.js (hunk 1 above)
Comment (orrpan) [reply devices.js:8219]: Naaah, It just happened when I tested and looked at the one above (mutliv4) and used the same.

**Comment by orrpan** [comment]
> I don't see any axis here (e.g. `x_axis`), does this device actually support it?

Something happens at least, here they use it for all multisensors https://github.com/SmartThingsCommunity/SmartThingsPublic/blob/a650b9d970b52ae8cb817a1747f113b7070a1d26/devicetypes/smartthings/smartsense-multi-sensor.src/smartsense-multi-sensor.groovy#L211

### closed door
```log
Zigbee2MQTT:debug 2021-02-28 09:26:39: Received Zigbee message from 'Hall Farstu', type 'attributeReport', cluster 'manuSpecificSamsungAccelerometer', data '{"x_axis":34,"y_axis":-1079,"z_axis":18}' from endpoint 1 with groupID 0
```

### open door
```log
Zigbee2MQTT:debug 2021-02-28 09:26:49: Received Zigbee message from 'Hall Farstu', type 'attributeReport', cluster 'manuSpecificSamsungAccelerometer', data '{"x_axis":39,"y_axis":-1080,"z_axis":10}' from endpoint 1 with groupID 0
```

**Comment by Koenkk** [comment]
> > I don't see any axis here (e.g. `x_axis`), does this device actually support it?
> 
> Something happens at least, here they use it for all multisensors https://github.com/SmartThingsCommunity/SmartThingsPublic/blob/a650b9d970b52ae8cb817a1747f113b7070a1d26/devicetypes/smartthings/smartsense-multi-sensor.src/smartsense-multi-sensor.groovy#L211
> 
> ### closed door
> ```
> Zigbee2MQTT:debug 2021-02-28 09:26:39: Received Zigbee message from 'Hall Farstu', type 'attributeReport', cluster 'manuSpecificSamsungAccelerometer', data '{"x_axis":34,"y_axis":-1079,"z_axis":18}' from endpoint 1 with groupID 0
> ```
> 
> ### open door
> ```
> Zigbee2MQTT:debug 2021-02-28 09:26:49: Received Zigbee message from 'Hall Farstu', type 'attributeReport', cluster 'manuSpecificSamsungAccelerometer', data '{"x_axis":39,"y_axis":-1080,"z_axis":10}' from endpoint 1 with groupID 0
> ```

Can you provide the debug log what happens after this? I expect it to be published to MQTT and the values shouldn't be N/A anymore in the frontend

To enable debug logging set in `configuration.yaml`:
```yaml
advanced:
  log_level: debug
```

**Comment by orrpan** [comment]
```

Zigbee2MQTT:debug 2021-03-01 09:10:22: Received Zigbee message from 'Hall Farstu', type 'attributeReport', cluster 'manuSpecificSamsungAccelerometer', data '{"acceleration":0}' from endpoint 1 with groupID 0,
Zigbee2MQTT:info  2021-03-01 09:11:22: MQTT publish: topic 'zigbee2mqtt/Hall Farstu', payload '{"battery":78,"battery_low":false,"contact":true,"linkquality":76,"moving":false,"tamper":false,"temperature":21.07}',
Zigbee2MQTT:info  2021-03-01 09:10:22: MQTT publish: topic 'zigbee2mqtt/Hall Farstu', payload '{"battery":78,"battery_low":false,"contact":true,"linkquality":76,"moving":false,"tamper":false,"temperature":21.07}',
Zigbee2MQTT:debug 2021-03-01 09:11:22: Received Zigbee message from 'Hall Farstu', type 'attributeReport', cluster 'manuSpecificSamsungAccelerometer', data '{"x_axis":41,"y_axis":-1073,"z_axis":16}' from endpoint 1 with groupID 0,
Zigbee2MQTT:debug 2021-03-01 09:11:22: Received Zigbee message from 'Hall Farstu', type 'attributeReport', cluster 'manuSpecificSamsungAccelerometer', data '{"acceleration":0}' from endpoint 1 with groupID 0,
Zigbee2MQTT:debug 2021-03-01 09:12:22: Received Zigbee message from 'Hall Farstu', type 'attributeReport', cluster 'manuSpecificSamsungAccelerometer', data '{"x_axis":41,"y_axis":-1073,"z_axis":16}' from endpoint 1 with groupID 0,
Zigbee2MQTT:info  2021-03-01 09:12:22: MQTT publish: topic 'zigbee2mqtt/Hall Farstu', payload '{"battery":78,"battery_low":false,"contact":true,"linkquality":76,"moving":false,"tamper":false,"temperature":21.07}',
Zigbee2MQTT:debug 2021-03-01 09:12:23: Received Zigbee message from 'Hall Farstu', type 'attributeReport', cluster 'manuSpecificSamsungAccelerometer', data '{"acceleration":0}' from endpoint 1 with groupID 0,
Zigbee2MQTT:info  2021-03-01 09:12:23: MQTT publish: topic 'zigbee2mqtt/Hall Farstu', payload '{"battery":78,"battery_low":false,"contact":true,"linkquality":76,"moving":false,"tamper":false,"temperature":21.07}',
Zigbee2MQTT:debug 2021-03-01 09:13:23: Received Zigbee message from 'Hall Farstu', type 'attributeReport', cluster 'manuSpecificSamsungAccelerometer', data '{"x_axis":41,"y_axis":-1073,"z_axis":16}' from endpoint 1 with groupID 0,
Zigbee2MQTT:info  2021-03-01 09:13:23: MQTT publish: topic 'zigbee2mqtt/Hall Farstu', payload '{"battery":78,"battery_low":false,"contact":true,"linkquality":76,"moving":false,"tamper":false,"temperature":21.07}',
Zigbee2MQTT:debug 2021-03-01 09:13:23: Received Zigbee message from 'Hall Farstu', type 'attributeReport', cluster 'manuSpecificSamsungAccelerometer', data '{"acceleration":0}' from endpoint 1 with groupID 0,
Zigbee2MQTT:info  2021-03-01 09:13:23: MQTT publish: topic 'zigbee2mqtt/Hall Farstu', payload '{"battery":78,"battery_low":false,"contact":true,"linkquality":78,"moving":false,"tamper":false,"temperature":21.07}',
Zigbee2MQTT:debug 2021-03-01 09:14:23: Received Zigbee message from 'Hall Farstu', type 'attributeReport', cluster 'manuSpecificSamsungAccelerometer', data '{"x_axis":41,"y_axis":-1073,"z_axis":16}' from endpoint 1 with groupID 0,
Zigbee2MQTT:info  2021-03-01 09:14:23: MQTT publish: topic 'zigbee2mqtt/Hall Farstu', payload '{"battery":78,"battery_low":false,"contact":true,"linkquality":76,"moving":false,"tamper":false,"temperature":21.07}',
Zigbee2MQTT:debug 2021-03-01 09:14:24: Received Zigbee message from 'Hall Farstu', type 'attributeReport', cluster 'manuSpecificSamsungAccelerometer', data '{"acceleration":0}' from endpoint 1 with groupID 0,
Zigbee2MQTT:debug 2021-03-01 09:15:24: Received Zigbee message from 'Hall Farstu', type 'attributeReport', cluster 'manuSpecificSamsungAccelerometer', data '{"x_axis":41,"y_axis":-1073,"z_axis":16}' from endpoint 1 with groupID 0,
Zigbee2MQTT:info  2021-03-01 09:14:24: MQTT publish: topic 'zigbee2mqtt/Hall Farstu', payload '{"battery":78,"battery_low":false,"contact":true,"linkquality":76,"moving":false,"tamper":false,"temperature":21.07}',
Zigbee2MQTT:info  2021-03-01 09:15:24: MQTT publish: topic 'zigbee2mqtt/Hall Farstu', payload '{"battery":78,"battery_low":false,"contact":true,"linkquality":76,"moving":false,"tamper":false,"temperature":21.07}',
Zigbee2MQTT:debug 2021-03-01 09:15:24: Received Zigbee message from 'Hall Farstu', type 'attributeReport', cluster 'manuSpecificSamsungAccelerometer', data '{"acceleration":0}' from endpoint 1 with groupID 0,
Zigbee2MQTT:debug 2021-03-01 09:16:24: Received Zigbee message from 'Hall Farstu', type 'attributeReport', cluster 'manuSpecificSamsungAccelerometer', data '{"x_axis":41,"y_axis":-1073,"z_axis":16}' from endpoint 1 with groupID 0,
Zigbee2MQTT:info  2021-03-01 09:16:24: MQTT publish: topic 'zigbee2mqtt/Hall Farstu', payload '{"battery":78,"battery_low":false,"contact":true,"linkquality":76,"moving":false,"tamper":false,"temperature":21.07}',
Zigbee2MQTT:debug 2021-03-01 09:16:25: Received Zigbee message from 'Hall Farstu', type 'attributeReport', cluster 'manuSpecificSamsungAccelerometer', data '{"acceleration":0}' from endpoint 1 with groupID 0,
Zigbee2MQTT:info  2021-03-01 09:16:25: MQTT publish: topic 'zigbee2mqtt/Hall Farstu', payload '{"battery":78,"battery_low":false,"contact":true,"linkquality":78,"moving":false,"tamper":false,"temperature":21.07}',
Zigbee2MQTT:debug 2021-03-01 09:17:25: Received Zigbee message from 'Hall Farstu', type 'attributeReport', cluster 'manuSpecificSamsungAccelerometer', data '{"x_axis":41,"y_axis":-1073,"z_axis":16}' from endpoint 1 with groupID 0,
Zigbee2MQTT:info  2021-03-01 09:17:25: MQTT publish: topic 'zigbee2mqtt/Hall Farstu', payload '{"battery":78,"battery_low":false,"contact":true,"linkquality":78,"moving":false,"tamper":false,"temperature":21.07}',
Zigbee2MQTT:debug 2021-03-01 09:17:25: Received Zigbee message from 'Hall Farstu', type 'attributeReport', cluster 'manuSpecificSamsungAccelerometer', data '{"acceleration":0}' from endpoint 1 with groupID 0,
Zigbee2MQTT:info  2021-03-01 09:17:25: MQTT publish: topic 'zigbee2mqtt/Hall Farstu', payload '{"battery":78,"battery_low":false,"contact":true,"linkquality":76,"moving":false,"tamper":false,"temperature":21.07}',
Zigbee2MQTT:info  2021-03-01 09:18:25: MQTT publish: topic 'zigbee2mqtt/Hall Farstu', payload '{"battery":78,"battery_low":false,"contact":true,"linkquality":76,"moving":false,"tamper":false,"temperature":21.07}',
Zigbee2MQTT:debug 2021-03-01 09:18:25: Received Zigbee message from 'Hall Farstu', type 'attributeReport', cluster 'manuSpecificSamsungAccelerometer', data '{"x_axis":41,"y_axis":-1073,"z_axis":16}' from endpoint 1 with groupID 0,
Zigbee2MQTT:debug 2021-03-01 09:18:26: Received Zigbee message from 'Hall Farstu', type 'attributeReport', cluster 'manuSpecificSamsungAccelerometer', data '{"acceleration":0}' from endpoint 1 with groupID 0,
Zigbee2MQTT:info  2021-03-01 09:18:26: MQTT publish: topic 'zigbee2mqtt/Hall Farstu', payload '{"battery":78,"battery_low":false,"contact":true,"linkquality":76,"moving":false,"tamper":false,"temperature":21.07}',
Zigbee2MQTT:debug 2021-03-01 09:19:26: Received Zigbee message from 'Hall Farstu', type 'attributeReport', cluster 'manuSpecificSamsungAccelerometer', data '{"x_axis":41,"y_axis":-1073,"z_axis":16}' from endpoint 1 with groupID 0,
Zigbee2MQTT:info  2021-03-01 09:19:26: MQTT publish: topic 'zigbee2mqtt/Hall Farstu', payload '{"battery":78,"battery_low":false,"contact":true,"linkquality":78,"moving":false,"tamper":false,"temperature":21.07}',
Zigbee2MQTT:debug 2021-03-01 09:19:26: Received Zigbee message from 'Hall Farstu', type 'attributeReport', cluster 'manuSpecificSamsungAccelerometer', data '{"acceleration":0}' from endpoint 1 with groupID 0,
Zigbee2MQTT:info  2021-03-01 09:19:26: MQTT publish: topic 'zigbee2mqtt/Hall Farstu', payload '{"battery":78,"battery_low":false,"contact":true,"linkquality":76,"moving":false,"tamper":false,"temperature":21.07}',
Zigbee2MQTT:debug 2021-03-01 09:20:26: Received Zigbee message from 'Hall Farstu', type 'attributeReport', cluster 'manuSpecificSamsungAccelerometer', data '{"x_axis":41,"y_axis":-1073,"z_axis":16}' from endpoint 1 with groupID 0,
Zigbee2MQTT:info  2021-03-01 09:20:26: MQTT publish: topic 'zigbee2mqtt/Hall Farstu', payload '{"battery":78,"battery_low":false,"contact":true,"linkquality":78,"moving":false,"tamper":false,"temperature":21.07}',
Zigbee2MQTT:debug 2021-03-01 09:20:27: Received Zigbee message from 'Hall Farstu', type 'attributeReport', cluster 'manuSpecificSamsungAccelerometer', data '{"acceleration":0}' from endpoint 1 with groupID 0,
Zigbee2MQTT:info  2021-03-01 09:20:27: MQTT publish: topic 'zigbee2mqtt/Hall Farstu', payload '{"battery":78,"battery_low":false,"contact":true,"linkquality":76,"moving":false,"tamper":false,"temperature":21.07}',
Zigbee2MQTT:debug 2021-03-01 09:21:27: Received Zigbee message from 'Hall Farstu', type 'attributeReport', cluster 'manuSpecificSamsungAccelerometer', data '{"x_axis":41,"y_axis":-1073,"z_axis":16}' from endpoint 1 with groupID 0,
Zigbee2MQTT:debug 2021-03-01 09:21:27: Received Zigbee message from 'Hall Farstu', type 'attributeReport', cluster 'manuSpecificSamsungAccelerometer', data '{"acceleration":0}' from endpoint 1 with groupID 0,
Zigbee2MQTT:info  2021-03-01 09:21:27: MQTT publish: topic 'zigbee2mqtt/Hall Farstu', payload '{"battery":78,"battery_low":false,"contact":true,"linkquality":78,"moving":false,"tamper":false,"temperature":21.07}',
Zigbee2MQTT:info  2021-03-01 09:21:27: MQTT publish: topic 'zigbee2mqtt/Hall Farstu', payload '{"battery":78,"battery_low":false,"contact":true,"linkquality":76,"moving":false,"tamper":false,"temperature":21.07}',
Zigbee2MQTT:debug 2021-03-01 09:22:27: Received Zigbee message from 'Hall Farstu', type 'attributeReport', cluster 'manuSpecificSamsungAccelerometer', data '{"x_axis":41,"y_axis":-1073,"z_axis":16}' from endpoint 1 with groupID 0,
Zigbee2MQTT:info  2021-03-01 09:22:27: MQTT publish: topic 'zigbee2mqtt/Hall Farstu', payload '{"battery":78,"battery_low":false,"contact":true,"linkquality":78,"moving":false,"tamper":false,"temperature":21.07}',
Zigbee2MQTT:debug 2021-03-01 09:22:28: Received Zigbee message from 'Hall Farstu', type 'attributeReport', cluster 'manuSpecificSamsungAccelerometer', data '{"acceleration":0}' from endpoint 1 with groupID 0,
Zigbee2MQTT:debug 2021-03-01 09:23:28: Received Zigbee message from 'Hall Farstu', type 'attributeReport', cluster 'manuSpecificSamsungAccelerometer', data '{"x_axis":41,"y_axis":-1073,"z_axis":16}' from endpoint 1 with groupID 0,
Zigbee2MQTT:info  2021-03-01 09:22:28: MQTT publish: topic 'zigbee2mqtt/Hall Farstu', payload '{"battery":78,"battery_low":false,"contact":true,"linkquality":76,"moving":false,"tamper":false,"temperature":21.07}',
Zigbee2MQTT:info  2021-03-01 09:23:28: MQTT publish: topic 'zigbee2mqtt/Hall Farstu', payload '{"battery":78,"battery_low":false,"contact":true,"linkquality":76,"moving":false,"tamper":false,"temperature":21.07}',
Zigbee2MQTT:debug 2021-03-01 09:23:28: Received Zigbee message from 'Hall Farstu', type 'attributeReport', cluster 'manuSpecificSamsungAccelerometer', data '{"acceleration":0}' from endpoint 1 with groupID 0
```

**Comment by Koenkk** [comment]
@orrpan are you sure the changes are correctly applied, becuase I don't see the axis values being published. Since it works for @ssmall I assume the change itself is correct.
